src/main/scala/ai/privado/passes/SQLParser.scala

```diff
@@ -76,34 +76,41 @@ class SQLParser(cpg: Cpg, projectRoot: String) extends ForkJoinParallelCpgPass[S
     sqlFile.close()
 
     sqlQueries
-      .flatMap(queryWthLine => {
-
+      .map(queryWthLine => {
         val query           = queryWthLine._1
         val queryLineNumber = queryWthLine._2
         try {
           SQLParser.parseSqlQuery(query) match {
-            case Some((queryName, tableName, columns)) =>
+            case Some(tmp) =>
               // Have added tableName in name key
               // Have added columns in value key
               // findMatchingIndices(lines, query).headOption.getOrElse(-1)
-              val sqlQueryNode =
-                NewSqlQueryNode()
-                  .code(query)
-                  .name(tableName)
-                  .fullName(query)
-                  .value(columns.mkString(","))
-                  .lineNumber(queryLineNumber)
-              builder.addNode(sqlQueryNode)
-              Some(sqlQueryNode)
-            case None => None
+              val res = tmp.map { case (_, tableName, columns) =>
+                val columnList = columns.mkString(", ")
```

Comment:
Any specific reason to change the mkString from `,` to `, `?

Reply:
Updated